libnavigation-core/src/main/java/com/mapbox/navigation/core/accounts/BillingController.kt

```diff
@@ -432,7 +438,6 @@ internal class BillingController(
 
     private fun handlerError(error: BillingServiceError) {
         when (error.code) {
-            BillingServiceErrorCode.INVALID_SKU_ID,
```

Comment:
> 'when' expression on enum is recommended to be exhaustive, add 'INVALID_SKU_ID' branch or 'else' branch instead

Are we good removing `INVALID_SKU_ID` branch?

Reply:
`BillingServiceErrorCode#INVALID_SKU_ID` does not exist in Common SDK v21. If you see this warning in the IDE you might need to sync Gradle to refresh the sources.

Reply:
I'd swear I navigated to the source code and saw it. Let me re-check.

Reply:
It doesn't show up now. Gradle chache...

Reply:
:magic_wand: